offchain/Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [workspace]
 members = [
   "advance-runner",
+  "authority-claimer",
```

Comment:
We also need to add the new service to the Dockerfile, [docker-bake.hcl](https://github.com/cartesi/rollups-node/blob/next/docker-bake.hcl), CI workflow, and README. This can be done later, but please make sure there is an issue for that.